.github/workflows/finalize-tests.yml

```diff
@@ -187,7 +187,7 @@ jobs:
           persist-credentials: false
       - name: "Cleanup docker"
         run: ./scripts/ci/cleanup_docker.sh
-      - name: Free up disk space
+      - name: "Free up disk space"
```

Comment:
YAML does not need quotes except if you carry special chars that need to be escaped :-D

Reply:
Added as a safety hatch. Thanks!